### PR TITLE
Repair server updates

### PR DESCRIPTION
**Pull request by dansisson**

### CODE REVIEW

**Comment by dansisson** [comment]
If this PR needs approval, or if the commits need to be pulled into the private repo, contact @alkohli. Thanks.

File: azure-stack/hci/deploy/deployment-install-os.md [hunk 1]
```diff
@@ -122,6 +122,9 @@ Follow these steps to configure the operating system using SConfig:
 
 1. Clean all the non-OS drives for each server that you intend to deploy. Remove any virtual media that have been used when installing the OS. Also validate that no other root drives exist.
 
+    > [!NOTE]
+    > This step does not apply when doing a server repair task.
+
```
Comment (alkohli) [inline azure-stack/hci/deploy/deployment-install-os.md:127]: Minor tweak - This step doesn't apply to a server repair operation.

**Review by Karl-WE — CHANGES_REQUESTED** [review]
I am requesting to add a link on netbios naming convention on the word Netbios. 

This convention also applies to the Cluster name
https://learn.microsoft.com/en-us/windows/win32/sysinfo/computer-names

**Comment by alkohli** [comment]
Hello @Karl-WE  PRs from Dan are coming for a different workflow. I have requested him to rebuild this PR and also add the link to netbios naming convention.

File: azure-stack/hci/manage/repair-server.md [hunk 2]
```diff
@@ -143,6 +143,9 @@ Follow these steps on another server that is a member of the same Azure Stack HC
     Repair-Server -Name "< Name of the new server>" -LocalAdminCredential $Cred
     ```
 
+    > [!NOTE]
+    > The server name must be the [NetBIOS name](/windows/win32/sysinfo/computer-names) This convention also applies to the cluster name.
+
```
Comment (alkohli) [inline azure-stack/hci/manage/repair-server.md:148]: Change this to: > The server name must be the [NetBIOS name](/windows/win32/sysinfo/computer-names).

**Comment by alkohli** [comment]
Looks good. Signing off.

#sign-off

**Pull request merged**